service/app/validators/email_validator.rb

```diff
@@ -0,0 +1,7 @@
+class EmailValidator < ActiveModel::EachValidator
+	def validate_each(record, attribute, value)
+		unless value =~ /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use a guard clause instead of wrapping the code inside a conditional expression. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)